tests/core/fs/workspacefs/test_file_history.py

```diff
@@ -0,0 +1,65 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pytest
+
+from parsec.core.fs.workspacefs.versioning_helpers import VersionLister
+from parsec.core.types import FsPath
+
+
+@pytest.mark.trio
+async def test_file_history(alice_workspace):
+    sync_by_id = alice_workspace.sync_by_id
+    entry = alice_workspace.get_workspace_entry()
+    wid = entry.id
+
+    # Empty workspace
+    await sync_by_id(wid)
```

Comment:
I don't thing this is needed: alice_workspace fixture should ensure the user & workspaces manifests has been synced

However it seems it's not the case:
https://github.com/Scille/parsec-cloud/blob/c73f6a84755f894baa2ffa668dfe70dd84d18cfe/tests/core/fs/workspacefs/conftest.py#L9-L14

So I think the fixture (and also bob_workspace fixture) should be modified to also do a `alice_user_fs.sync()` before returning